Runtime/JSCallbackArgs.cs

```diff
@@ -1,51 +1,57 @@
+using System;
 using System.Runtime.InteropServices;
 using static NodeApi.JSNativeApi.Interop;
 
 namespace NodeApi;
 
-public class JSCallbackArgs
+public readonly ref struct JSCallbackArgs
```

Comment:
>readonly ref struct [](http://example.com/codeflow?start=7&length=19)

This is a good change! I can see how it makes the perf 15% faster because we effectively avoid extra memory allocation per each call.